openedx/core/djangoapps/coursetalk/helpers.py

```diff
@@ -16,7 +16,8 @@ def get_coursetalk_course_key(course_key):
     :return: CourseTalk course key
     :rtype: str
     """
-    return '{0.org}_{0.course}'.format(course_key)
+
+    return '{0.CANONICAL_NAMESPACE}:{0.org}+{0.course}+{0.run}'.format(course_key)
```

Comment:
Will this break existing course talk integrations? I'm nervous about the implications of changing the key.

Is it possible to view the existing integration on your sandbox? It's shown on course about pages, right? It would be good to confirm that it is still working. In fact, it might be worth changing the existing integration to render using the new fragment.